### PR TITLE
protos: Add definitions to telemetry.vehicle_data Field enums

### LINKED ISSUE #106: Vehicle data enum definitions
The Field enums for the proto vehicle data are without definitions. This is hard for those of us who use this data. We need to understand it, to work with it correctly.

Please, add descriptions to these enums. It will help us all – those who want to use this thing and not as close to it.

I tried to fix it, with guesses and GPT4's help. I'll send a pull request. Maybe it's a start.

P.S Thanks for making great products, my model s is the greatest thing I have ever owned   😊

### PR DESCRIPTION
**Pull request by nicksherron**
# Description

Adds commented definitions for the vehicle data proto Field enums

Fixes #106 

## Type of change

Please select all options that apply to this change:

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Documentation update

# Checklist:

Confirm you have completed the following steps:

- [x] My code follows the style of this project.
- [x] I have performed a self-review of my code.
- [x] I have made corresponding updates to the documentation.
- [ ] I have added/updated unit tests to cover my changes.
- [ ] I have added/updated integration tests to cover my changes.

### CODE REVIEW

File: protos/vehicle_data.proto [hunk 1]
```diff
@@ -7,170 +7,498 @@ import "google/protobuf/timestamp.proto";
 option go_package = "github.com/teslamotors/fleet-telemetry/protos";
 
 enum Field {
+  // Unknown field, default value
```
Comment (agbpatro) [inline protos/vehicle_data.proto:10]: can you run `make generate-protos`?

Comment (nicksherron) [reply protos/vehicle_data.proto:10]: Done! Sorry for the late response, I just saw your comment.

File: protos/vehicle_error.pb.go [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.28.1
-// 	protoc        v4.25.1
+// 	protoc-gen-go v1.32.0
+// 	protoc        v3.21.12
```
[the remark this answers is not in the document]
Comment (nicksherron) [reply protos/vehicle_error.pb.go]: Thanks for your patience @agbpatro I think it's been updated properly

**Comment by nicksherron** [comment]
@vmallet I agree with you and will close this. My main goal was to get some clarity on the descriptions and the pr was more or less a way to get feedback if my "guesses" were correct.

**Comment by nicksherron** [comment]
@vmallet I agree with you and will close this. My main goal was to get some clarity on the descriptions and the pr was more or less a way to get feedback if my "guesses" were correct.

**Pull request closed without merging**